### PR TITLE
Update build documentation for Rust Installation and Building Clamor

### PR DESCRIPTION
**Pull request by satendrapalsingh**
Add some details to the rust installation section and mention WSL for developers using Windows.

### CODE REVIEW

File: docs/docs/contribute/code/building-clamor.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+# Building Clamor
+
+*Before you start, ensure you've [set up your development environment](../getting-started/#development-environment).*
+
+This guide will outline the process to build [Clamor](https://github.com/fragcolor-xyz/clamor) from the sources.
+
+## Windows
+
+!!! note
+    Fragcolor doesn't yet officially support building and developing Clamor on the Windows platform. Please use Linux or Mac for Clamor. You may use the Linux/Mac [instructions](#linuxmac) on Windows with [WSL](https://docs.microsoft.com/en-us/windows/wsl/) but your mileage may vary and results are not guaranteed. Clamor is based on [Substrate](https://substrate.io/) so you could also check out their [support for Windows](https://docs.substrate.io/v3/getting-started/windows-users/).
```
Comment (sinkingsugar) [inline docs/docs/contribute/code/building-clamor.md:15]: We don't support exactly because Substrate doesn't work on windows. So this is a bit misleading. Not our fault ;)

**Comment by Kryptos-FR** [comment]
Agreeing with @sinkingsugar remark, we can maybe rephrase the part about Windows not being supported. Otherwise, LGTM.

File: docs/docs/contribute/code/building-clamor.md (hunk 1 above)
Comment (satendrapalsingh) [reply docs/docs/contribute/code/building-clamor.md:15]: Oh, alright! Yes I didn't notice that. I'll rephrase it.

Comment (satendrapalsingh) [reply docs/docs/contribute/code/building-clamor.md]: I have rephrased the note. Edit commit added (no rebase/ force-push).

Comment (satendrapalsingh) [reply docs/docs/contribute/code/building-clamor.md]: Please review and mark as resolved if satisfied.

**Review by Kryptos-FR — APPROVED** [review]

**Comment by sinkingsugar** [comment]
I was very confused about this issue.. this should be in Clamor readme..... nothing to do with chainblocks.

**Comment by satendrapalsingh** [comment]
I have moved added the build-clamor instructions to Clamor's README itself ([issue](https://github.com/fragcolor-xyz/clamor/issues/51) , [PR](https://github.com/fragcolor-xyz/clamor/pull/65)).

Will remove the [building-clamor page](https://docs.fragcolor.xyz/contribute/code/building-clamor/) as part of issue #228

**Comment by satendrapalsingh** [comment]
Building-clamor documentation removed from `docs.fragcolor.xyz` via [this commit](https://github.com/fragcolor-xyz/chainblocks/pull/238/commits/86cb7e73f32162df0f6d494f8ed3dddeb0ecee2b) and being added to Clamor README via [this commit](https://github.com/fragcolor-xyz/clamor/pull/65/commits/bb3bb03925fee8cc2e5ef69afa2dda1e49dbdbfa).
